v20.04/Dockerfile.multiarch

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/owncloud/base:20.04@sha256:0a37898f4df9a6951ccb3ce1d702b5aa188012d4bf00beaa018062cb548fa2d8
+FROM docker.io/owncloud/base:22.04@sha256:f29ec067a2c67bbfff2cad95cd24b38610830001aaf8ee189c94ba51547ef0a6
```

Comment:
The top folder is still called `v20.04`
I will make my own PR to change to Ubuntu 22.04 and Freexian PHP 7.4